### PR TITLE
RDoc-1510 Proofreading

### PR DESCRIPTION
**Pull request by morhilai**

### CODE REVIEW

File: Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/lets-encrypt-certificates.markdown [hunk 1]
```diff
@@ -19,33 +19,33 @@ When there are 30 days left until expiration, RavenDB will initiate the certific
 Once the renewed certificate is obtained, [it will be replaced](../../../server/security/authentication/certificate-renewal-and-rotation) in all the nodes of the cluster without needing to shut down any server.
 
 {WARNING: Warning} 
-Automatic renewals of certificates is available only if you obtained your certificate using the Setup Wizard and got your free RavenDB domain. It doesn't work for self-obtained certificates, even if issued by Let's Encrypt.
+Automatic renewals of certificates are available only if you obtained your certificate using the Setup Wizard and got your free RavenDB domain. Independently obtained certificates will not renew automatically, even if issued by Let's Encrypt.
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/lets-encrypt-certificates.markdown]: Make singular 
=>
Automatic renewal of a certificate is available only if you obtained your certificate...
-or-
Automatic certificate renewal is available only if you obtained your certificate...

File: Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/lets-encrypt-certificates.markdown [hunk 2]
```diff
@@ -19,33 +19,33 @@ When there are 30 days left until expiration, RavenDB will initiate the certific
...
+* There are only 3 days left for the expiration of the certificate. In this case, the cluster will complete the operation without the node which is down. **When bringing that node up, the certificate must be replaced manually.**
 
 During the process you will receive alerts in the studio and in the logs indicating the status of the operation and any errors if they occur. The alerts are displayed for each node independently.
 
 ## Manual Renewal
 
-You may initiate the renewal process manually by going to the certificate view in the studio and clicking `Renew` on the server certificate. It will trigger the same certificate replacement process which was described in `Automatic Renewal`.
+You may initiate the renewal process manually by going to the certificate view in the studio and clicking `Renew` on the server certificate. It will trigger the same certificate replacement process which was described in [Automatic Renewal](http://localhost:54391/article/4.1/csharp/server/security/authentication/lets-encrypt-certificates#automatic-renewal).
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/lets-encrypt-certificates.markdown]: Replace link with "../../../server/security/authentication/lets-encrypt-certificates#automatic-renewal"

File: Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/lets-encrypt-certificates.markdown [hunk 3]
```diff
@@ -19,33 +19,33 @@ When there are 30 days left until expiration, RavenDB will initiate the certific
...
-If a node is down and you click `Renew`, the cluster will complete the operation without the node which is down. When bringing that node up, the certificate must be replaced manually.
+If a node is down and you click `Renew`, the cluster will complete the operation without the node that is down. **When bringing that node up, the certificate must be replaced manually.**
 
 
 ## Updating DNS records
 
 Updating DNS records for your domain can be acheived by running the Setup Wizard again or by using a dedicated page at the RavenDB website.
 
-The [Customers Portal](https://customers.ravendb.net) allows to easily edit DNS records which are associated with your license.
+The [Customers Portal](https://customers.ravendb.net) allows you to easily edit DNS records which are associated with your license.
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/lets-encrypt-certificates.markdown]: Although correct, it is better to use:
=> 
You can easily edit the DNS records which are associated with your license using the [Customers Portal](https://customers.ravendb.net)

File: Documentation/4.1/Raven.Documentation.Pages/start/installation/setup-wizard.markdown [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Installation : Setup Wizard Walkthrough
 
-We want to make it as easy as possible for you to start RavenDB with a valid trusted certificate from the very beginning and to stay secure through your entire application lifecycle, starting from early stages of development, ending on production and day-to-day usage.
+We want to make it as easy as possible for you to start RavenDB with a valid, trusted certificate from the very beginning. We want you to stay secure throughout your application lifecycle, starting with the early stages of development and all the way to production and day-to-day usage.
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/start/installation/setup-wizard.markdown]: This sentence needs refactoring. Let's talk.

File: Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/lets-encrypt-certificates.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/lets-encrypt-certificates.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/lets-encrypt-certificates.markdown (hunk 2 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/lets-encrypt-certificates.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/lets-encrypt-certificates.markdown (hunk 3 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/server/security/authentication/lets-encrypt-certificates.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/start/installation/setup-wizard.markdown (hunk 4 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/start/installation/setup-wizard.markdown]: Done

**Pull request merged**